Google.PowerShell/Properties/AssemblyInfo.cs

```diff
@@ -38,5 +38,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.0.0.10")]
-[assembly: AssemblyFileVersion("1.0.0.10")]
+[assembly: AssemblyVersion("1.0.1.0")]
```

Comment:
Are we bumping the 3rd # in because we are adding modules?

Reply:
We are bumping because this module now also works on PowerShell Core.